cmd/backend/add_param/add-param.go

```diff
@@ -0,0 +1,62 @@
+package add_param
+
+/*
+$ go run main.go backend add-param --name "YourParameterName" --value "YourParameterValue" --type "String"
```

Comment:
so wouldn't we run `dlp-cli backend add-param --name "YourParamName" --value "YourParameterValue" --type "String"` when this is deployed?

Reply:
yea absolutely, lemme change that. this is jus the command i used for testing.